pumas_stochastic_collect_tau.F90

```diff
@@ -44,197 +41,223 @@ module pumas_stochastic_collect_tau
 ! Assume 6 microns for each...
 real(r8), parameter :: m1 = 4._r8/3._r8*pi*rhow*(6.e-6_r8)**3
 
+!$acc declare create(knn,cutoff_id,mmean,diammean,medge,diamedge)
 
 !===============================================================================
 contains
 !===============================================================================
 
-      
 subroutine calc_bins    
 
-real(r8) :: DIAM(ncdp)
-real(r8) :: X(ncdp)
-real(r8) :: radsl(ncdp)
-real(r8) :: radl(ncd)
-integer  :: L, lcl  
-real(r8) :: kkfac
+  implicit none
+
```

Comment:
This is unnecessary as it is declared above at the module level

Reply:
Thanks and I have removed it.